js/client-chat.js

```diff
@@ -1247,6 +1247,11 @@
 					if (!this.tournamentBox.parseMessage(row.slice(1), row[0] === 'tournaments')) break;
 					// fallthrough in case of unparsed message
```

Comment:
This is going to cause problems when you're changing what the fallthrough is, isn't it?